lib/session/session_test.go

```diff
@@ -28,10 +28,22 @@ import (
 
 	"github.com/gravitational/trace"
 	"github.com/jonboulle/clockwork"
-	. "gopkg.in/check.v1"
+	"github.com/stretchr/testify/require"
 )
 
-func TestSessions(t *testing.T) { TestingT(t) }
+func TestSessions(t *testing.T) {
+	utils.InitLoggerForTests(testing.Verbose())
+	s := NewSessionSuite(t)
+
+	t.Cleanup(func() {
```

Comment:
```go
t.Cleanup(t.tearDownSuite)
```

Reply:
That doesn't exist in gotest. I'm not using gocheck.

Reply:
We're moving away from that and it doesn't have a concept of suites. SessionSuite is just a non special struct containing state.

Reply:
And nothing is stopping you from adding a method on it - it does not run against testing package and it does not imply gocheck.

Reply:
Seems a pretty uneeded to typedef a new type just to add a method while an anon function does just fine to me anyway.